modules/cudf/src/series.ts

```diff
@@ -811,6 +811,27 @@ export class AbstractSeries<T extends DataType = any> {
       this._col.fill(new Scalar({type: this.type, value}), begin, end, memoryResource));
   }
 
+  /**
+   * Repeats a Series n times, returning a new Series.
+   *
+   * @param repeats The number of times to repeat this.
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * // Float64Series
+   * Series.new([1, 2, 3]).repeat(2) // [1, 2, 3, 1, 2, 3]
+   * // StringSeries
+   * Series.new(["foo", "bar", "test"]).repeat(2) // ["foo", "bar", "test", "foo", "bar", "test"]
+   * // Bool8Series
+   * Series.new([true, true, true]).repeat(2) // [true, false, false, true, false, false]
+   * ```
+   */
+  repeat(repeats: T['scalarType'], memoryResource?: MemoryResource): Series<T> {
+    return this.__construct(this._col.repeat(repeats, memoryResource));
```

Comment:
Looks like the Column method has this signature:
```typescript
repeat(value: Scalar<T>, begin?: number, end?: number, memoryResource?: MemoryResource)
```

Which seems incompatible with how it's being called here?

Reply:
I'm clearly not using `repeat` in the implementation below, I'll remove this stuff from this PR.